boards/arm/96b_stm32_sensor_mez/96b_stm32_sensor_mez.yaml

```diff
@@ -0,0 +1,9 @@
+identifier: 96b_stm32_sensor_mez
+name: 96B_STM32_SENSOR_MEZ
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+supported:
+  - rtc
```

Comment:
I think there are much more you can mention here (gpio, spi, ..)

Reply:
Ack